R/class-workbook-wrappers.R

```diff
@@ -457,6 +472,7 @@ wb_copy_cells <- function(
     as_ref    = FALSE,
     transpose = FALSE
 ) {
+  # TODO maybe consider copying cells with dims? instead of relying on `wb_data`
```

Comment:
The reason why I built this around `wb_data()` is that (a) I had it around since I used it in mschart, here and in pivot tables and (b) that is is possible to reference a sheet or a region to copy things over. The entire `wb_copy_cells()` thing is one I have never seen anybody using, but it was fun to implement and is somewhat robust.